### ISSUE TITLE
PageSippet: set/remove editable loses existing elements on save

### ISSUE THREAD

**Issue by fashxp** [pimcore/pimcore#6992]
Steps to reproduce:

```
$doc = Document::getById(99);
$doc->setEditable("somekey", "somedata");    // it doesn't check if existing editables are already loaded, so overwrites them ...
$doc->save();
```

see also https://github.com/pimcore/pimcore/pull/6982

**Comment by jdreesen** [comment]
This can be closed since #7001 got merged.

**Comment by dvesh3** [comment]
fixed by #7001
